understanding/22/focus-appearance-minimum.html

```diff
@@ -224,10 +224,10 @@ <h4>Unusual shapes and gradients</h4>
                     <figcaption> The focused button with the non-contrasting areas removed, showing that it is a thick indicator that meets the requirements.</figcaption>
                 </figure>
 
-                <p>If an inline link is broken over two or more lines, the indicator can be split between them. In this case the link element is still treated as a single control.</p>
+                <p>If an inline link is broken over two lines, some methods of outline are treated differently by browsers. Using CSS, an outline will either wrap each part of the link separately, or wrap the whole link as one perimeter. Each part of the link will pass or fail as though they were separate links. If a border is used the indicator can be split between them. When using a border the thickness may need to be increased to compensate.</p>
```

Comment:
suggest adding commas for better readability

`If a border is used, the indicator can be split between them. When using a border, the thickness may need to be increased to compensate.`

Reply:
done.